nginx.tmpl

```diff
@@ -147,6 +147,12 @@ upstream {{ $upstream_name }} {
 {{/* Get the VIRTUAL_PROTO defined by containers w/ the same vhost, falling back to "http" */}}
 {{ $proto := trim (or (first (groupByKeys $containers "Env.VIRTUAL_PROTO")) "http") }}
 
+{{/* Get the NETWORK_ACCESS defined by containers w/ the same vhost, falling back to "external" */}}
+{{ $network_tag := or (first (groupByKeys $containers "Env.NETWORK_ACCESS")) "external" }}
+
+{{/* Get the NETWORK_ACCESS defined by containers w/ the same vhost, falling back to "external" */}}
+{{ $network_tag := or (first (groupByKeys $containers "Env.NETWORK_ACCESS")) "external" }}
```

Comment:
Am I crazy, or is this block duplicated?

Reply:
You're absolutely right!  This PR was way behind master and was conflicting, so I rebased it and squashed the commit, and must have duplicated it in the process.  Thanks for reporting it, @josephholsten, I'll send a bugfix PR right away.

Reply:
Pushed fix in #977